apps/cyberstorm-remix/app/p/components/ReportPackage/ReportPackageForm.tsx

```diff
@@ -127,7 +126,11 @@ export function ReportPackageForm(
             // placeholder="Please select..."
             value={formInputs.reason}
             onChange={(value) => {
-              updateFormFieldState({ field: "reason", value: value });
+              if (!isStringOneOf(value, reportReasonValues)) {
```

Comment:
Offtopic: requiring this sort of check sort of hints that the typing of the component _might_ have room for improvement. I've no suggestions though and it's not worth spending time (mine or yours) on now. But having to do this sort of thing in each select element in the project seems like noise that should be avoided if possible. This comment is based on assumption that this check is only to satisfy TypeScript and we don't expect it to actually happen.